desktop/src/main/java/bisq/desktop/main/dao/governance/proposals/ProposalsView.java

```diff
@@ -223,6 +231,20 @@ protected void activate() {
         updateViews();
     }
 
+    private void maybeShowDAOTestingFeedbackWindow() {
+        String testingPopupKey = "daoTestingFeedbackPopup";
+        if (DontShowAgainLookup.showAgain(testingPopupKey)) {
+            UserThread.runAfter(() -> {
+                if (sortedList.size() > 0 && sortedList.stream().map(proposalsListItem -> proposalsListItem.getProposal().getTxId())
```

Comment:
Better to use `myProposalListService.getList()' - this gives you your own proposals and test with `periodService.isTxInCorrectCycle(tx.getBlockHeight(), daoStateService.getChainHeight())` if it the proposal is in the current cycle.